packages/model-viewer/src/three-components/Reticle.ts

```diff
@@ -53,28 +53,31 @@ export default class Reticle extends Object3D {
     return this._hitMatrix;
   }
 
+  reset() {
```

Comment:
What is the intended purpose of this new public API? (please add docs)

Reply:
Done.